DataPlotly/gui/layout_item_gui.py

```diff
@@ -125,36 +141,38 @@ def show_properties(self):
         """
         Shows the plot properties panel
         """
-        self.panel = DataPlotlyPanelWidget(mode=DataPlotlyPanelWidget.MODE_LAYOUT, message_bar=self.message_bar)
+        selected_plot_index = self.plot_list.currentRow()
+        if selected_plot_index >= 0:
```

Comment:
Just a minor thing, but instead of making all the rest of this function an indented block, it would be nicer to just return early:

    if not selected_plot_index:
        return

Reply:
Changed!